dhis-2/dhis-support/dhis-support-db-migration/src/main/java/org/hisp/dhis/db/migration/config/FlywayConfig.java

```diff
@@ -62,8 +62,9 @@ public Flyway flyway( DhisConfigurationProvider configurationProvider, DataSourc
             Boolean.parseBoolean( configurationProvider.getProperty( FLYWAY_OUT_OF_ORDER_MIGRATION ) ) );
         classicConfiguration.setIgnoreMissingMigrations( true );
         classicConfiguration.setIgnoreFutureMigrations( false );
-        classicConfiguration.setGroup( false );
+        classicConfiguration.setGroup( true );
```

Comment:
@luciano-fiandesio  Group ensures the "migration group" is executed in a single transaction. This would be ideal for instance to rollback to a clean state in case of migration failures. 
But we have introduced certain non-transaction statements in our migrations. We can enable "Mixed" mode so that it wraps in transaction on a best effort basis.